.github/workflows/package.yml

```
@@ -5,35 +5,35 @@ on:
     branches: [ "develop"]
     paths-ignore:
      - target/**
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 5

(indentation)